documentation.md

```diff
@@ -569,3 +569,63 @@ response:
     # on the actual value of it
     returned_uuid: null
 ```
+
+## Persistent cookies
+
+Tavern uses
+[requests](http://docs.python-requests.org/en/master/api/#requests.request)
+under the hood, and uses a persistent `Session` for each test. This means that
+cookies are propagated forward to further stages of a test. Cookies can also be
+required to pass a test. For example, say we have a server that returns a cookie
```

Comment:
Mention where the source for this test is?

Reply:
Which source? This is just an example of a server that would behave like that

Reply:
`example/cookies/`?

Reply:
That is just an example implementation of the server in python - it doesn't really relate to the documentation. There aren't implementations for any of the other examples